### PR TITLE
dynamic get battery name

### PR DESCRIPTION
**Pull request by hellwen**

### CODE REVIEW

**Comment by hellwen** [comment]
battery name in my thinkpad x260 is BAT1, so i change the default BAT0 to dynamic get from BAT_DIR

File: libqtile/widget/battery.py [hunk 1]
```diff
@@ -61,9 +61,10 @@ class _Battery(base._TextBox):
     """Base battery class"""
 
     filenames = {}
+    # ACPI name of a battery, usually BAT0
+    battery_name = 'BAT0'
 
     defaults = [
-        ('battery_name', 'BAT0', 'ACPI name of a battery, usually BAT0'),
```
Comment (tych0) [inline libqtile/widget/battery.py]: Can we just s/BAT0/get_battery_name()/ here? That way we will still get all the docs autogenerated for this from config.

Comment (hellwen) [reply libqtile/widget/battery.py]: maybe, i will try it

Comment (hellwen) [reply libqtile/widget/battery.py]: _get_battery_name() replaced BAT0

**Comment by tych0** [comment]
LGTM, thanks!

**Pull request merged**